### ISSUE TITLE
Accessing Core is forbidden 

### ISSUE THREAD

**Issue by TimoZacherl** [NamelessMC/Nameless#1081]
### Template:
* What kind of issue is it (Question, Bug, etc.)?
Bug
* In which version of NamelessMC did it occur (1.x.x)?
1.0.20
* How would you describe the issue?
Getting: 

Forbidden
You don't have permission to access /admin/core on this server.

when i am trying to access the core page at the admincp, every other page works fine.

**Comment by baxandrei** [comment]
Are You using script in an subdirectory?

**Comment by TimoZacherl** [comment]
Yes there is a file and when i directly Type the url .../admin/core.PHP in but that is not how it should be i guess

**Comment by samerton** [comment]
To be honest, I don't know what the problem could be. It'll be down to some server configuration related issue, or a custom modification to Nameless somewhere.

Can you edit the file `.htaccess` in the Nameless root directory (for example /var/www/html on a VPS or public_html on a web host) and check the contents match [this](https://raw.githubusercontent.com/NamelessMC/Nameless/master/.htaccess) exactly?

**Comment by baxandrei** [comment]
I have same problem, if you use VPS, use that in putty as root: `sudo chown apache:apache -R /var/www`

**Comment by TimoZacherl** [comment]
The .htaccess is the same as yours and i am using webspace from strato

**Comment by samerton** [comment]
Unfortunately I don't know what the problem is. 

I'm not sure this is a Nameless problem, there is no reason why access to /admin/core would result in a 403 whereas the rest of the site and the AdminCP works.

Would you be able to test out [version 2](https://github.com/NamelessMC/Nameless/archive/v2.zip) in a subfolder, eg yoursite.com/v2, just to see if the AdminCP -> Core tab has the same problem there?

**Comment by TimoZacherl** [comment]
On version 2 it works just fine i think i would stay on this one. But i just wonder why i get this error on v1

**Comment by baxandrei** [comment]
You don`t get any error on error_log?

**Comment by TimoZacherl** [comment]
No not a single error
